<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7SBKKJ1" failures="22" tests="87" name="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" time="300.307" errors="0" timestamp="2024-09-13T10:59:02 IST" skipped="0">
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Click_On_Report" time="7.346"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Click_On_CROSS_SELL_CONVERSION_REPORT" time="4.313"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Zone_And_Region_Wise_Tab" time="48.170"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Dealer_Parent_Group_Wise_Tab" time="57.326"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="WorkShop_Wise_Tab" time="60.727"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Day_Wise_Tab" time="55.262"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Service_Type_Wise_Tab" time="59.050"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_DealerParentWise" time="1.011">
    <failure message="expected [6360] but found [3248]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [6360] but found [3248]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_DealerParentWise(CrossSellConversion_Report_SMR.java:533)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_DealerParentWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_DealerParentWise" time="0.000">
    <failure message="expected [3976] but found [3054]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3976] but found [3054]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_DealerParentWise(CrossSellConversion_Report_SMR.java:539)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_DealerParentWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnectedLeads_Column_ZoneRegionWise_With_DealerParentWise" time="0.000">
    <failure message="expected [598] but found [82]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [598] but found [82]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotConnectedLeads_Column_ZoneRegionWise_With_DealerParentWise(CrossSellConversion_Report_SMR.java:545)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotConnectedLeads_Column_ZoneRegionWise_With_DealerParentWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_DealerParentWise" time="0.000">
    <failure message="expected [1786] but found [112]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [1786] but found [112]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_DealerParentWise(CrossSellConversion_Report_SMR.java:551)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_DealerParentWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_ZoneRegionWise_With_DealerParentWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReadyToPurchaseLeads_Column_ZoneRegionWise_With_DealerParentWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCompletedLeads_Column_ZoneRegionWise_With_DealerParentWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_ZoneRegionWise_With_DealerParentWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_WorkshopWise" time="0.000">
    <failure message="expected [6360] but found [3248]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [6360] but found [3248]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_WorkshopWise(CrossSellConversion_Report_SMR.java:581)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_WorkshopWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_WorkshopWise" time="0.015">
    <failure message="expected [3976] but found [3054]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3976] but found [3054]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_WorkshopWise(CrossSellConversion_Report_SMR.java:587)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_WorkshopWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnectedLeads_Column_ZoneRegionWise_With_WorkshopWise" time="0.000">
    <failure message="expected [598] but found [82]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [598] but found [82]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotConnectedLeads_Column_ZoneRegionWise_With_WorkshopWise(CrossSellConversion_Report_SMR.java:593)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotConnectedLeads_Column_ZoneRegionWise_With_WorkshopWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_WorkshopWise" time="0.000">
    <failure message="expected [1786] but found [112]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [1786] but found [112]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_WorkshopWise(CrossSellConversion_Report_SMR.java:599)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_WorkshopWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_ZoneRegionWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReadyToPurchaseLeads_Column_ZoneRegionWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCompletedLeads_Column_ZoneRegionWise_With_WorkshopWise" time="1.009"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_ZoneRegionWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_DayWise" time="0.000">
    <failure message="expected [6360] but found [3248]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [6360] but found [3248]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_DayWise(CrossSellConversion_Report_SMR.java:630)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_DayWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_DayWise" time="0.000">
    <failure message="expected [3976] but found [3054]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3976] but found [3054]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_DayWise(CrossSellConversion_Report_SMR.java:636)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_DayWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnectedLeads_Column_ZoneRegionWise_With_DayWise" time="0.000">
    <failure message="expected [598] but found [82]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [598] but found [82]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotConnectedLeads_Column_ZoneRegionWise_With_DayWise(CrossSellConversion_Report_SMR.java:642)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotConnectedLeads_Column_ZoneRegionWise_With_DayWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_DayWise" time="0.000">
    <failure message="expected [1786] but found [112]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [1786] but found [112]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_DayWise(CrossSellConversion_Report_SMR.java:648)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_DayWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_ZoneRegionWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReadyToPurchaseLeads_Column_ZoneRegionWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCompletedLeads_Column_ZoneRegionWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_ZoneRegionWise_With_DayWise" time="0.015"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_ServiceTypeWise" time="0.000">
    <failure message="expected [6360] but found [3249]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [6360] but found [3249]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:678)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_TotalInterestedLeads_Column_ZoneRegionWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_ServiceTypeWise" time="0.000">
    <failure message="expected [3976] but found [3055]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3976] but found [3055]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:684)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotAttemptedLeads_Column_ZoneRegionWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnectedLeads_Column_ZoneRegionWise_With_ServiceTypeWise" time="0.000">
    <failure message="expected [598] but found [82]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [598] but found [82]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotConnectedLeads_Column_ZoneRegionWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:690)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotConnectedLeads_Column_ZoneRegionWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_ServiceTypeWise" time="0.000">
    <failure message="expected [1786] but found [112]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [1786] but found [112]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:696)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_ConnectedFollowUpLeads_Column_ZoneRegionWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_ZoneRegionWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReadyToPurchaseLeads_Column_ZoneRegionWise_With_ServiceTypeWise" time="0.008"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCompletedLeads_Column_ZoneRegionWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_ZoneRegionWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_DealerParentWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_DealerParentWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnectedLeads_Column_DealerParentWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_DealerParentWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_DealerParentWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReadyToPurchaseLeads_Column_DealerParentWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCompletedLeads_Column_DealerParentWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_DealerParentWise_With_WorkshopWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_DealerParentWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_DealerParentWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnected_Column_DealerParentWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_DealerParentWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_DealerParentWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReadyToPurchaseLeads_Column_DealerParentWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCompletedLeads_Column_DealerParentWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_DealerParentWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_DealerParentWise_With_ServiceTypeWise" time="0.000">
    <failure message="expected [3248] but found [3249]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3248] but found [3249]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_TotalInterestedLeads_Column_DealerParentWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:822)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_TotalInterestedLeads_Column_DealerParentWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_DealerParentWise_With_ServiceTypeWise" time="0.016">
    <failure message="expected [3054] but found [3055]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3054] but found [3055]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotAttemptedLeads_Column_DealerParentWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:828)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotAttemptedLeads_Column_DealerParentWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnectedLeads_Column_DealerParentWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_DealerParentWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_DealerParentWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReadyToPurchaseLeads_Column_DealerParentWise_With_ServiceTypeWise" time="1.013"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCompletedLeads_Column_DealerParentWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_DealerParentWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_WorkshopWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_WorkshopWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnectedLeads_Column_WorkshopWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_WorkshopWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_WorkshopWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReadyToPurchaseLeads_Column_WorkshopWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCompletedLeads_Column_WorkshopWise_With_DayWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_WorkshopWise_With_DayWise" time="0.001"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_WorkshopWise_With_ServiceTypeWise" time="1.007">
    <failure message="expected [3248] but found [3249]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3248] but found [3249]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_TotalInterestedLeads_Column_WorkshopWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:919)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_TotalInterestedLeads_Column_WorkshopWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_WorkshopWise_With_ServiceTypeWise" time="0.000">
    <failure message="expected [3054] but found [3055]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3054] but found [3055]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotAttemptedLeads_Column_WorkshopWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:925)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotAttemptedLeads_Column_WorkshopWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnectedLeads_Column_WorkshopWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_WorkshopWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_WorkshopWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReaduToPurchaseLeads_Column_WorkshopWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchseCompletedLeads_Column_WorkshopWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_WorkshopWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_TotalInterestedLeads_Column_DayWise_With_ServiceTypeWise" time="0.000">
    <failure message="expected [3248] but found [3249]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3248] but found [3249]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_TotalInterestedLeads_Column_DayWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:967)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_TotalInterestedLeads_Column_DayWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotAttemptedLeads_Column_DayWise_With_ServiceTypeWise" time="0.000">
    <failure message="expected [3054] but found [3055]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3054] but found [3055]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at merge_SMR_Central_Zone.CrossSellConversion_Report_SMR.Compare_NotAttemptedLeads_Column_DayWise_With_ServiceTypeWise(CrossSellConversion_Report_SMR.java:973)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Compare_NotAttemptedLeads_Column_DayWise_With_ServiceTypeWise -->
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_NotConnectedLeads_Column_DayWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ConnectedFollowUpLeads_Column_DayWise_With_ServiceTypeWise" time="0.003"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ProposalSharedLeads_Column_DayWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_ReadyToPurchaseLeads_Column_DayWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCompletedLeads_Column_DayWise_With_ServiceTypeWise" time="0.000"/>
  <system-out/>
  <testcase classname="merge_SMR_Central_Zone.CrossSellConversion_Report_SMR" name="Compare_PurchaseCancelledLeads_Column_DayWise_With_ServiceTypeWise" time="4.015"/>
  <system-out/>
</testsuite> <!-- merge_SMR_Central_Zone.CrossSellConversion_Report_SMR -->
